README.md

```diff
@@ -187,24 +187,30 @@ $ curl -X DELETE "localhost:8080/delete_all"
 
 ### Query Endpoints
 
-These endpoints match those used by the Grafana Simple JSON datasource.
+These endpoints match those used by the [Grafana Simple JSON datasource](https://grafana.com/grafana/plugins/grafana-simple-json-datasource/).
 
 #### GET /search
 
-Responds with a JSON array containing the selectable query elements.
+Responds with a JSON array containing selectable values of an event field (e.g. `jdk.ObjectAllocationSample.objectClass`) specified in the JSON body's `target` field. Used to define available selections for [dashboard variables](https://grafana.com/docs/grafana/v7.5/variables/).
```

Comment:
The heading here is still `GET /search` but the example shows a `POST` and the description is talking about a request body. Is the `GET` still supported or did `POST` wholly replace it?

Is `target` now a required request attribute or can I leave it out and have that behave the same way it used to?

Reply:
Oh that's totally my mistake. It should be entirely `POST` now as specified by the plugin. I will fix it.

Grafana will send the `POST` request with a body as `{ "target": "some-target"}`.